### DISCUSSION TITLE
Why does `Layer.dependsOn()` not only have `vararg` argument?

### DISCUSSION THREAD

**Discussion by WildOrangutan** [LemonAppDev/konsist#1477] [Report a Problem]
### Tell us what version of Konsist you use.

0.16.1

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I'm looking at API for `Layer.deependsOn()`. Why does it have a `layer` for first argument? I think it makes API a bit annoying to use?

```kotlin
fun Layer.dependsOn(
    layer: Layer,
    vararg layers: Layer,
): Unit

fun Layer.doesNotDependOn(
    layer: Layer,
    vararg layers: Layer,
): Unit
```

### Actual behaviour

Current usage:

```kotlin
val layer: Layer = TODO()
val otherLayers = listOf<Layer>(TODO())
layer.dependsOn(otherLayers.first(), otherLayers.drop(1))
```

### Expected behaviour

What would be nice:

```kotlin
val layer: Layer = TODO()
val otherLayers = listOf<Layer>(TODO())
layer.dependsOn(otherLayers)
```

**Comment by igorwojda** [comment]
Good point. It will be included this in next version of Konsist (0.17.0).

**Comment by nataliapeterwas** [comment]
Konsist `0.17.0` has been released! This release includes this improvement :) See full release log: [Release log](https://github.com/LemonAppDev/konsist/releases/tag/v0.17.0)
